### PR TITLE
Sbd conversion language updates

### PR DESCRIPTION
**Pull request by TimCliff**
This PR updates the language on the SBD -> STEEM conversion page to make the risks of SBD conversion clearer to the user.

Note - the translation file updates are untested.

### CODE REVIEW

File: app/components/elements/ConvertToSteem.jsx [hunk 1]
```diff
@@ -44,9 +44,9 @@ class ConvertToSteem extends React.Component {
                     <div className="small-12 columns">
                         <h1>{translate('convert_to_LIQUID_TOKEN')}</h1>
                         <p>{translate('DEBT_TOKEN_will_be_unavailable')}.</p>
+                        <p>{translate('DEBT_TOKEN_conversions_have_risk')}.</p>
                         {/* using <FormattedMessage /> because nested html tag in values doesn't want to be rendered properly in translate() */}
-                        <p><FormattedMessage id="your_existing_DEBT_TOKEN_are_liquid_and_transferable" values={{ link: <i>{translate("buy_or_sell")}</i> }} /></p>
-                        <p>{translate('this_is_a_price_feed_conversion')}.</p>
+                        <p><FormattedMessage id="your_existing_DEBT_TOKEN_are_liquid_and_transferable" values={{ link: <i>{translate("currency_market")}</i> }} /></p>
```
Comment (roadscape) [inline app/components/elements/ConvertToSteem.jsx]: Why the change to `currency_market` from `buy_or_sell`?

**Comment by TimCliff** [comment]
Hi @roadscape - it looked like the "buy or sell" was outdated. All of the links that I saw to the internal exchange for SBD <-> STEEM were either labeled "Market" or "Internal Market".

**Comment by TimCliff** [comment]
Hi @roadscape - do you need me to make any changes for this? Based on my understanding, renaming it to "Internal Market" made more sense, but if you would prefer I go back to using "buy or sell", I can make the change.
